scripts/commit_classifier.py

```diff
@@ -148,6 +150,13 @@ def __init__(
                 "hg::https://hg.mozilla.org/mozilla-central", git_repo_dir
             )
 
+        self.revision = None
+        if phabricator_deployment is not None and diff_id is not None:
```

Comment:
nit: Checking `phabricator_deployment` as an assertion will help catch missing argument if the this get called directly in the future:
```suggestion
        if diff_id is not None:
            assert phabricator_deployment is not None
```